.vsts-ci.yml

```diff
@@ -53,6 +53,14 @@ variables:
 stages:
 - stage: Build
   jobs:
+  - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/main')) }}:
```

Comment:
This may be a nit (because I haven't looked at how long installer builds take) but&hellip; For a number of larger repos @wtgodbe and I enabled `OneLocBuild` for, we limited ourselves to scheduled builds (once per day). I believe the loc team recommends that somewhere❔

Reply:
Interesting, the SDK repo was doing every build so I just copied the logic from there.  There's only 40 files at the moment and they rarely change so there isn't a need to parse them every build...Do you have an example of how aspnet does it on a schedule?

Reply:
> Do you have an example of how aspnet does it on a schedule?

We use a separate pipeline (that only exists internally) based on the YAML at https://github.com/dotnet/aspnetcore/blob/main/.azure/pipelines/localization.yml. The logic makes things run _only_ for scheduled or manual builds in the pipeline.

Reply:
Thanks. Pushed an update to move the build to a different file. Not sure how to test that I did it right and we'll get it running on the schedule. Any suggestions?

Reply:
You could ask Core-Eng to set up a new isntaller-localization pipeline pointing to where the YAML will live. Fine to have the default be the 'main' branch even though this isn't merged yet. Then run a manual build and confirm the generated loc project looks like you expect.
But, I probably wouldn't worry about it. You're not doing anything here that can break the main pipeline. Worst case, you may need to do a follow up after the first scheduled build.